src/collections/_documentation/accounts/saml2.md

```diff
@@ -0,0 +1,123 @@
+<md:KeyDescriptor use="signing">
+<ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
+<ds:X509Data>
+<ds:X509Certificate>
+MIIDpDCCAoygAwIBAgIGAV3jWPYbMA0GCSqGSIb3DQEBCwUAMIGSMQswCQYDVQQGEwJVUzETMBEG A1UECAwKQ2FsaWZvcm5pYTEWMBQGA1UEBwwNU2FuIEZyYW5jaXNjbzENMAsGA1UECgwET2t0YTEU MBIGA1UECwwLU1NPUHJvdmlkZXIxEzARBgNVBAMMCmRldi01MTcyNDkxHDAaBgkqhkiG9w0BCQEW DWluZm9Ab2t0YS5jb20wHhcNMTcwODE1MDA0MzA0WhcNMjcwODE1MDA0NDAzWjCBkjELMAkGA1UE BhMCVVMxEzARBgNVBAgMCkNhbGlmb3JuaWExFjAUBgNVBAcMDVNhbiBGcmFuY2lzY28xDTALBgNV BAoMBE9rdGExFDASBgNVBAsMC1NTT1Byb3ZpZGVyMRMwEQYDVQQDDApkZXYtNTE3MjQ5MRwwGgYJ KoZIhvcNAQkBFg1pbmZvQG9rdGEuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA o7OQ51PQdel7doiZ4tXRBGoB+6TgUZLqA0JGMcRCBsS6ApMaxho7kc+urC3I6CrL4pw4Dw6RGXY4 bENVbhAJo46U1kN2QYnnyV/e+GqJgKQK4qGllS2dfGsMeiyj9QXocP+h13VNeeHb0xBAo7qZTGxW aNn16+OGuw02JQY5F7MRT0UKNNVnuXjp3z8NdUHqoxMmdo0TTgOXb9BgmgYSxkRn6iBF0tG3tpig lUt2CJUKsLPgq/dSMlce+XUIwdDErebfqtB3dQ9O7iIp5ZpynPUkcEMmjMznK/pptHzvoUY8sWBN XJcqEsXT83RTNTdoo0zpUDjaWMVH/su6seGDqwIDAQABMA0GCSqGSIb3DQEBCwUAA4IBAQCOqhYY w8cmqvJyihgY8Ir50Zg6EQqlLS0whNuUOgQDHEMxSbkC+/dfuOSIeS9Ps/7sg5J4wcDs/enrKOMS BD3r9jH7E6KNEoleRv8AWRXlwQnIOajPQKFSTUQDF5Z/aDPc4SvrAQvXE+KggviwA80xxyGDf64L qYeCG3WiShHgBOVIpXuItSQTRZFjIvZRDJyawXpYfuSj2jxNdg+ChuhF4+baHfKwYvP6Z6YuoQvM aRjS00xKN7h4Pq1KtCfxyQC+WobCErYojEasRS/JUS4cCAkwtUKcha63HIayTCdgrfaKW9aEwACk IXWNkhXDx03WbSxgxRMidkA1eADs30r1
+</ds:X509Certificate>
+</ds:X509Data>
+</ds:KeyInfo>
+</md:KeyDescriptor>
+<md:NameIDFormat>
+urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
+</md:NameIDFormat>
+<md:NameIDFormat>
+urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress
+</md:NameIDFormat>
+<md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://dev-517249.oktapreview.com/app/sentry/exkf25e40eapgLtXx0h7/sso/saml"/>
+<md:SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://dev-517249.oktapreview.com/app/sentry/exkf25e40eapgLtXx0h7/sso/saml"/>
+</md:IDPSSODescriptor>
+</md:EntityDescriptor>
+```
+
+### Using Provider Data
+This registration method is the most involved, and requires matching up the data fields from an IdP. The easiest way to accomplish this is to look for the values in a metadata file such as the one provided above.
+
+{% asset saml2-provider-data.png %}
+
+Based on the XML file example above, the field values to plug in are as follows:
+
+* `EntityID`
+* `X509Certificate`
+* `SingleSignOnService`
+
+In this example, the `SingleLogoutService` isn’t provided by the IdP, and is treated as an optional value within Sentry.
+
+
+## 3. Map IdP Attributes
+
+{% include components/alert.html
+  title="Note on field names"
+  content="Metadata field names can vary from one provider to another. For example, Microsoft Azure AD refers to these very metadata fields as `Claims`, while Okta refers to them as `Attributes`. Similarly, one platform might use `user.email`, while another vendor uses `emailaddress`."
```

Comment:
IMO the pink on red is a little hard to read. What do you think about bolding these terms instead? **Claims**, **Attributes**, etc.